src/Utils.php

```diff
@@ -25,7 +25,7 @@ public static function caselessRemove(array $keys, array $data): array
         }
 
         foreach ($data as $k => $v) {
```

Comment:
Do we also want to add `@param array<string, mixed> $data` on this method?

Reply:
No. That would be a breaking change. I'd be happy to do this in a possible future 3.x version, though.